content/sensu-go/6.1/observability-pipeline/observe-events/events.md

```diff
@@ -120,7 +120,32 @@ Sensu events contain:
 ## Use event data
 
 Observability data in events is a powerful tool for automating monitoring workflows.
-For example, you can [reduce alert fatigue][23] by filtering events based on the event `occurrences` attribute.
+For example, you can use the [`state` attribute][36] to provide handlers with more information about changes in check status and [reduce alert fatigue][23] by filtering events based on the event `occurrences` attribute.
+
+### State attribute
+
+The `state` event attribute adds meaning to the check status:
+
+- `passing` means the check status is `0` (OK).
+- `failing` means the check status changed from 0 to non-zero (WARNING or CRITICAL).
+- `flapping` means a rapid change in check result status (determined based on the [low and high flap thresholds][37] set in the check attributes).
```

Comment:
```suggestion
- `flapping` means a rapid change in check result status (determined based per-check [low and high flap thresholds][37] attributes).
```

Reply:
I'm not sure that rapid is the right word here. Rather, flapping conveys an unsteady state where the check is not settling on success or failure, in aggregate, over the last 21 executions.